PypeS/pypepad.py

```diff
@@ -276,7 +276,7 @@ def GetSuggestionsList(self,word):
             scriptindex = self.text_input.search('vmtk',self.wordIndex[0],backwards=1)
             moduleName  = self.text_input.get( scriptindex,scriptindex+' wordend' )
             try:
-                exec('import '+moduleName)
+                exec('from . import '+moduleName)
                 exec('scriptObjectClassName =  '+moduleName+'.'+moduleName)
```

Comment:
I wanted to highlight this section (and the import section directly above) since **this is a major change to the structure of the code.** 

The combination of lines 244-249 and 251-252 decouples the explicit definition in the first line of a vmtkScript between the files name and the name of the class inside. Instead, the scriptObjectClassName is identified by searching attributes within a file and selecting the first class that it comes across, dynamically creating the scriptObject from a call to `getattr(module, scriptObjectClassName)()
`
The benefits of this change are:

1. This is a much **more "pythonic"** way of handling dynamic object imports. It takes advantage of the core language features (everything is a first-class object, and self introspection) which make python so powerful for the wide variety of use cases. 
2. It is much **more transparent and readable** - this codeblock is a core method of the library, and the use of exec() obfuscates the meaning and importance of the section. 
3. We should see **significant performance improvements** over the analogous `exec()` call. The reasons are two-fold:
    1. In the modified form, the interpreter has all the required variables available in it's local scope, and is able to optimize execution of the module import and class instantiation . This is in direct contrast to the full-halt enabled whenever an `exec() `call is made, a process where the interpreter has to compile the import string and any associated object to bytecode (a time consuming process) before actually doing any of the loading and class instantiation in an isolated scope whose objects must be backported into the calling functions locals() / globals() namespace. 
    2. When executing a function within the scope of an` exec() `call, the (cyclicly executing) python garbage collector cannot identify the scope which any objects created within the  `exec() `namespace were created in. Effectively, this means that the garbage collector does not run on these objects! Objects created within an exec() call have to explicitly call the` __del__ `deconstructor, or the garbage collector will not remove the objects from memory. 

_Preferably, we would remove all uses of the exec() function from the library._ There are very few use-cases of exec which cannot be more optimally and more transparently executed with python standard library functions. 

More info at the following links:
- http://lucumr.pocoo.org/2011/2/1/exec-in-python/
- http://stackoverflow.com/a/29456463
- https://late.am/post/2012/04/30/the-exec-statement-and-a-python-mystery.html